### ISSUE TITLE
Clang 5.0 test error on POWER8

### ISSUE THREAD

**Issue by khuck** [STEllAR-GROUP/phylanx#247]
http://ktau.nic.uoregon.edu:8020/builders/ppc64le-clang5-release/builds/283/steps/Test%20Phylanx/logs/stdio

Specific test that fails:
```
22/74 Test #22: tests.unit.primitives.column_slicing ..............................***Failed    1.85 sec
<unknown>
terminate called after throwing an instance of 'hpx::exception_list'
  what():  HPX(unknown_error)
Base command is "/home/users/khuck/buildbot/slaves/phylanx/ppc64le-clang5-release/build/tools/buildbot/build-centaur-ppc64le-Linux-clang/phylanx-Release/bin/column_slicing_test --hpx:threads=1 --hpx:localities=1"
Executing command: /home/users/khuck/buildbot/slaves/phylanx/ppc64le-clang5-release/build/tools/buildbot/build-centaur-ppc64le-Linux-clang/phylanx-Release/bin/column_slicing_test --hpx:threads=1 --hpx:localities=1 --hpx:node=0
```
GDB output:

```bash
terminate called after throwing an instance of 'hpx::exception_list'
  what():  HPX(unknown_error)

Program received signal SIGABRT, Aborted.
0x00003fffb3d8eb18 in raise () from /lib64/libc.so.6
Missing separate debuginfos, use: debuginfo-install glibc-2.17-157.el7_3.4.ppc64le libicu-50.1.2-15.el7.ppc64le
(gdb) where
#0  0x00003fffb3d8eb18 in raise () from /lib64/libc.so.6
#1  0x00003fffb3d90c9c in abort () from /lib64/libc.so.6
#2  0x00003fffb40fd7c4 in __gnu_cxx::__verbose_terminate_handler ()
    at ../../../../libstdc++-v3/libsupc++/vterminate.cc:95
#3  0x00003fffb40fa0b4 in __cxxabiv1::__terminate (handler=<optimized out>)
    at ../../../../libstdc++-v3/libsupc++/eh_terminate.cc:47
#4  0x00003fffb40fa170 in std::terminate () at ../../../../libstdc++-v3/libsupc++/eh_terminate.cc:57
#5  0x00003fffb40fa078 in std::rethrow_exception (ep=...) at ../../../../libstdc++-v3/libsupc++/eh_ptr.cc:259
#6  0x00003fffb5d9cd00 in hpx::runtime_impl::rethrow_exception() ()
   from /home/users/khuck/buildbot/slaves/phylanx/ppc64le-clang5-release/build/tools/buildbot/build-centaur-ppc64le-Linux-clang/hpx-Release/lib/libhpx.so.1
#7  0x00003fffb5d9ce04 in hpx::runtime_impl::run(hpx::util::function<int (), false> const&) ()
   from /home/users/khuck/buildbot/slaves/phylanx/ppc64le-clang5-release/build/tools/buildbot/build-centaur-ppc64le-Linux-clang/hpx-Release/lib/libhpx.so.1
#8  0x00003fffb5d52088 in hpx::detail::run(hpx::runtime&, hpx::util::function<int (boost::program_options::variables_map&), false> const&, boost::program_options::variables_map&, hpx::runtime_mode, hpx::util::unique_function<void (), false>, hpx::util::unique_function<void (), false>) ()
   from /home/users/khuck/buildbot/slaves/phylanx/ppc64le-clang5-release/build/tools/buildbot/build-centaur-ppc64le-Linux-clang/hpx-Release/lib/libhpx.so.1
#9  0x00003fffb5d52910 in hpx::detail::run_or_start(bool, std::unique_ptr<hpx::runtime, std::default_delete<hpx::runtime> >, hpx::util::command_line_handling&, hpx::util::unique_function<void (), false>, hpx::util::unique_function<void (), false>) ()
   from /home/users/khuck/buildbot/slaves/phylanx/ppc64le-clang5-release/build/tools/buildbot/build-centaur-ppc64le-Linux-clang/hpx-Release/lib/libhpx.so.1
#10 0x00003fffb5d52e90 in hpx::detail::run_or_start(hpx::util::function<int (boost::program_options::variables_map&), false> const&, boost::program_options::options_description const&, int, char**, std::vector<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::allocator<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > > >&&, hpx::util::unique_function<void (), false>, hpx::util::unique_function<void (), false>, hpx::runtime_mode, bool) ()
   from /home/users/khuck/buildbot/slaves/phylanx/ppc64le-clang5-release/build/tools/buildbot/build-centaur-ppc64le-Linux-clang/hpx-Release/lib/libhpx.so.1
---Type <return> to continue, or q <return> to quit---q
```

**Comment by justwagle** [comment]
Fixed with #299 
Buildbot: http://ktau.nic.uoregon.edu:8020/builders/ppc64le-clang5-release/builds/329

**Comment by khuck** [comment]
Yup, looks good.  Thanks!
